doc/changes/names.inc

```diff
@@ -342,4 +342,6 @@
 
 .. _Hongjiang Ye: https://github.com/rubyyhj
 
+.. _Jeff Stout: https://megcore.nih.gov/index.php/Staff
```

Comment:
@jstout211 let me know if you want a different URL (or if this is not you!), I chose this from 1 minute of Google-fu

Reply:
@jstout211 I'm going to merge this PR but feel free to comment here if the name or URL should be changed